Maquetado/scss/details.scss

```diff
@@ -72,3 +82,43 @@
   font-size: 20px;
   line-height: 24px;
 }
+
+.background {
+  background-color: $wild-sand;
+}
+
+.list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 auto;
+  max-width: 1000px;
+}
+
+.list-book {
+  background-color: $white;
+  display: flex;
+  @extend %shadow;
+  flex-flow: column wrap;
+  margin: 20px;
```

Comment:
I think that only with padding is sufficient.

Reply:
The margin is to separate between books, otherwise they are all nearby